index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css.css">
+    <title>Maidan</title>
+  </head>
+  <body>
+    <div class="container">
+      <div id="search">
+        <h2>Search request</h2>
+        <textarea id="searchInputField" onkeyup="adjustInputFieldSize()" name="name" rows="1" cols="70"></textarea>
+      </div>
+      <div id="TestsAndResult">
+
+        <div class="wrapper">
+          <div id="tests">
+            <h2>Tests</h2>
+            <div id="testsList">
+            </div>
+          </div>
+
+          <div id="results">
+            <h2>Results</h2>
+            <div id="resultList">
+              <div class="result">
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div id="createTestButton">
+          <img src="resources/btn2.png" alt="">
+        </div>
```

Comment:
`img` buttons are not the best choice. It is really better to make them with styles.
Also, consider using `<button>` tags.